tests/unit/jobs/test_views.py

```diff
@@ -841,3 +855,111 @@ def test_get_submission_by_pk_when_user_is_neither_challenge_host_nor_submission
         response = self.client.get(self.url)
         self.assertEqual(response.data, expected)
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class ChallengeLeaderboardTest(BaseAPITestClass):
+
+    def setUp(self):
+        super(ChallengeLeaderboardTest, self).setUp()
+
+        self.dataset_split = DatasetSplit.objects.create(
+            name="Split 1",
+            codename="split1"
+        )
+
+        self.challenge_phase_split = ChallengePhaseSplit.objects.create(
+            challenge_phase=self.challenge_phase,
+            dataset_split=self.dataset_split,
+            leaderboard=self.leaderboard,
+            visibility=ChallengePhaseSplit.PUBLIC
+        )
+
+        self.submission = Submission.objects.create(
+            participant_team=self.participant_team,
+            challenge_phase=self.challenge_phase,
+            created_by=self.user1,
+            status="submitted",
+            input_file=self.challenge_phase.test_annotation,
+            method_name="Test Method",
+            method_description="Test Description",
+            project_url="http://testserver/",
+            publication_url="http://testserver/"
+        )
+
+        self.submission.is_public = True
+        self.submission.status = Submission.FINISHED
+        self.submission.save()
+
+        self.result_json = {
+            'score': 50,
+            'test-score': 50
```

Comment:
Just a minor change, can we have different value for `test-score`. I want to have different values for both the matrics, like if one is 50.0 then other can be 75.0 or any positive number other than 50.0.
Also, please add float values.